### PR TITLE
Fixing Arabic number issue

### PR DESCRIPTION
**Pull request by smahi**
This is an attempt to fix Arabic number displaying issue.

### CODE REVIEW

**Comment by djoudi** [comment]
@omaralalwi

**Comment by omaralalwi** [comment]
thanks, i will review ASAP and merge

File: src/Builders/PdfBuilder.php
Comment (omaralalwi) [inline src/Builders/PdfBuilder.php:1]: no need to pass config again here, we already passed in getPdfBuilder function in Gpdf class

File: junit.xml
Comment (omaralalwi) [inline junit.xml:1]: please clean unwanted changes

**Pull request merged**